Sources/Extensions/SwiftStdlib/CollectionExtensions.swift

```diff
@@ -188,3 +188,15 @@ public extension Collection where Element: FloatingPoint {
     }
 
 }
+
+// MARK: - Properties (Hashable)
+public extension Collection where Element: Hashable {
```

Comment:
Can you put this a level up and make it an extension of `Sequence`?

Reply:
Actually, I just realised `Sequence` doesn't have a `count`. You could still put it there, but you'd need to manually add each element to the set, and check before if it exists. e.g.
extension Sequence where Element: Hashable {
```Swift
extension Sequence where Element: Hashable {
    func containsDuplicates() -> Bool {
        var set = Set<Element>()
        for element in self {
            if !set.insert(element).inserted {
                return true
            }
        }
        return false
    }
}
```